README.md

```diff
@@ -1,5 +1,5 @@
-# CustomGento Tax Editor
-n98-magerun module for Magento 1 to edit the tax configuration.
+# Tax Editor for Magento 1
+Tax Editor for Magento 1 adds a n98-magerun command to automatically update tax rates. This is pretty useful for general tax rate changes in a country, which must be applied exactly at a specific date. It is possible to define a new rate for a range of tax rate IDs.
```

Comment:
`automatically` is not really correct. Please incorporate the feedback I gave to the M2 description. Please also change this in the website repository.